docs/features.md

```diff
@@ -0,0 +1,297 @@
+
+Default settings:
+
+```bash
+"API_GATEWAY ":{
+  "isEnabled": true,
+  "selectors":[
+    {
+      "type":"apiGroup",
+      "apiGroup":"gateway.kyma-project.io"
+    }
+  ]
+},
+```
+
+- **APPLICATIONS** – is used to show or hide the Applications view as well as to define which Custom Resource Definitions (CRD) are required for the view to be shown properly.
+  For the view to be shown, the feature must be enabled and all CRDs, defined in the selectors array, must exist in a cluster.
+
+Default settings:
+
+```bash
+"APPLICATIONS":{
+  "isEnabled": true,
+  "selectors":[
+    {
+      "type":"apiGroup",
+      "apiGroup":"applicationconnector.kyma-project.io"
+    }
+  ]
+},
+```
+
+- **BTP_CATALOG** – is used to show or hide the BTP Catalog view as well as to define which Custom Resource Definitions (CRD) are required for the view to be shown properly.
+  For the view to be shown, the feature must be enabled, and all CRDs, defined in the selectors array, must exist in a cluster.
+
+Default settings:
+
+```bash
+"BTP_CATALOG":{
+  "isEnabled": true,
+  "selectors":[
+    {
+      "type":"apiGroup",
+      "apiGroup":"services.cloud.sap.com"
+    }
+  ]
+},
+```
+
+- **CUSTOM_DOMAINS** – is responsible for displaying views of components such as DNS entries, DNS providers, gateways, issuers and certificates. Its functions can be visible only if all Custom Resource Defintions, defined in the selectors array, exist in a cluster.
+
+Default settings:
+
+```bash
+"CUSTOM_DOMAINS": {
+  "isEnabled": true,
+  "selectors": [
+    {
+      "type": "apiGroup",
+      "apiGroup": "dns.gardener.cloud"
+    }
+  ]
+},
+```
+
+- **EVENTING** – is used to show or hide the Eventing view as well as to define which Custom Resource Definitions (CRD) are required for the view to be shown properly.
+  For the view to be shown, the feature must be enabled and all CRDs, defined in the selectors array, must exist in a cluster.
+
+Default settings:
+
+```bash
+"EVENTING ":{
+  "isEnabled": true,
+  "selectors":[
+    {
+      "type":"apiGroup",
+      "apiGroup":"eventing.kyma-project.io"
+    }
+  ]
+},
+```
+
+- **KUBECONFIG_ID** – is used to configure URL to which Busola sends a request to download a kubeconfig file. If we add kubeconfigID={your ID} to the Busola URL then Busola tries to download the kubeconfig from the {kubeconfigUrl}/{yourID} and, if succedded, will add a cluster.
+  You can introduce the whole address in the kubeconfigUrl which will also be read by Busola.
+
+Default settings:
+
+```bash
+"KUBECONFIG_ID": {
+ "isEnabled": true,
+ "config": {
+   "kubeconfigUrl": "https://kyma-env-broker.cp.dev.kyma.cloud.sap/kubeconfig"
+ }
+},
+```
+
+- **LEGAL_LINKS** – is used to show or hide legal links. In config you can find the unchangeable keys (you cannot use legalDisclosure instead of legal-disclosure). The keys include the default link, which takes you to the default address, and a link that depends on your chosen language, for example:
```

Comment:
Maybe we should provide list of all possible links to configure?

Reply:
I will provide all the links in the example.